Bio/PDB/PDBIO.py

```diff
@@ -395,7 +395,7 @@ def save(self, file, select=_select, write_end=True, preserve_atom_numbering=Fal
                             # inconsequential if preserve_atom_numbering is True
                             atom_number += 1
 
-                if chain_residues_written:
+                if chain_residues_written and write_ter == True:
```

Comment:
As per the automated checks, style wise just use ``if chain_residues_written and write_ter:``